sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusMetricsProvider.cs

```diff
@@ -96,13 +96,13 @@ public async Task<long> GetMessageCountAsync()
             {
                 if (TimeToLogWarning())
                 {
-                    _logger.LogWarning($"Connection string does not have 'Manage Claim' for {entityName} '{_entityPath}'. Unable to determine active message count.", ex);
+                    _logger.LogWarning(ex, $"Connection string does not have 'Manage Claim' for {entityName} '{_entityPath}'. Unable to determine active message count.");
                 }
                 throw ex;
             }
             catch (Exception e)
             {
-                _logger.LogWarning($"Error querying for Service Bus {entityName} scale status: {e.Message}");
+                _logger.LogWarning(e, $"Error querying for Service Bus {entityName} scale , stack: {e.StackTrace}");
```

Comment:
Why do we include the StackTrace in the message?

Reply:
this is outdated, I fixed this by moving the exception to the first parameter.

Reply:
removed